cmd/dump.go

```diff
@@ -0,0 +1,78 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cmd
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/cobraext"
+	"github.com/elastic/elastic-package/internal/dump"
+	"github.com/elastic/elastic-package/internal/elasticsearch"
+)
+
+const dumpLongDescription = `Use this command as a exploratory tool to dump assets relevant for the package.`
+
+const dumpInstalledObjectsLongDescription = `Use this command to dump objects installed by Fleet as part of a package.
+
+Use this command as a exploratory tool to dump objects as they are installed by Fleet when installing a package. Dumped objects are stored in files as they are returned by APIs of the stack, without any processing.`
+
+func setupDumpCommand() *cobraext.Command {
+	dumpInstalledObjectsCmd := &cobra.Command{
+		Use:   "installed-objects",
+		Short: "Dump objects installed in the stack",
+		Long:  dumpInstalledObjectsLongDescription,
+		RunE:  dumpInstalledObjectsCmd,
+	}
+	dumpInstalledObjectsCmd.Flags().StringP(cobraext.DumpPackageFlagName, "p", "", cobraext.DumpPackageFlagDescription)
```

Comment:
I hope it isn't possible to install multiple package versions at once :)

Reply:
Well, if some day this is possible, I guess that the worst thing that can happen with current implementation is that the objects of all installed versions will be collected, what I would find fine. We can add an optional version filter if this day arrives.